deploy/katello.spec

```diff
@@ -116,8 +116,9 @@ Summary:    Common runtime components of %{name}
 Requires:       wget
 Requires:       curl
 Requires:       %{?scl_prefix}rubygem-katello
-Requires:       rubygem-hammer_cli 
+Requires:       rubygem-hammer_cli
 Requires:       rubygem-hammer_cli_foreman
+Requires:       rubygem-hammer_cli_foreman_docker
```

Comment:
Looks like this is handled here -- https://github.com/Katello/hammer-cli-katello/blob/master/rubygem-hammer_cli_katello.spec#L27